packages/react/src/components/Chip/Chip.mdx

```diff
@@ -0,0 +1,124 @@
+import {
+  Meta,
+  Canvas,
+  Story,
+  Controls,
+  Primary,
+  ArgTypes,
+} from '@storybook/blocks';
+import { Information } from '../../../../../docs-components';
+import * as ChipStories from './Chip.stories';
+import { Chip } from './index';
+
+<Meta of={ChipStories} />
+
+# Chip
+
+<Information text='beta' />
+
+`Chips` er små avrundede knapper som kan brukes til å filtrere og navigere gjennom innhold eller data. For eksempel kan de brukes til å filtrere kategorier i et søkeresultat, samt vise hvilke filter som er aktive. Ikke bruk Chip til statisk metadata for å tagge innholdselementer. Bruk heller [Tag](https://digdir.github.io/designsystem/?path=/docs/kjernekomponenter-tag--docs) -komponenten dersom informasjonen ikke er klikkbar.
+
+Chip eksisterer i ulike varianter:
+
+- `Chip` er et alternativ til `button`. Den brukes når handlingen(e) er direkte relatert til hovedinnholdet og når handlingen ikke krever like mye oppmerksomhet som vanlige knapper. Det kan for eksempel være å klikke på en Chip for så å bli sendt til en søkeresultatside med filtrering på denne kategorien.
+- `Chip` med `checkmark=true` er alternativer til vanlige `radiobuttons` og `checkboxer`. Brukes til filtrering av innhold og data, når det skal tilbys et valg mellom flere ulike opsjoner. Innholdet endres til å vise kun det som er valgt.
+- `Removable Chip` er valgte verdier som brukeren kan fjerne. For eksempel filter som har blitt aktivert i et søk som kan fjernes igjen, eller som har blitt valgt i en `multiselect`.
+
+## Props
+
+<Primary />
+<Controls />
+
+## Bruk
+
+<Information text='token' />
+
+Chip-komponenten består av flere komponenter. Disse er:
+
+- Chip
+- Chip.Removable
+- Chip.Group
+
+Som standard så rendres Chip som et`<button>` element, men dette kan overstyres ved å bruke `as` egenskapen, `<Chip as="a">`. Dette vil gjøre at en Chip rendres som en anchor-tag istedenfor button.
+
+Hvordan komponentene som nevnes over brukes kan du se i eksemplene under.
+
+### Chip
+
+Denne kodeblokken representerer en enkel Chip med teksten "Nynorsk".
+
+```tsx
+<Chip onClick={handleOnClick}>Nynorsk</Chip>
+```
+
+<Canvas of={ChipStories.Default} />
+
+### Chip med sjekkmerkeikon
+
+Denne kodeblokken representerer en Chip med et sjekkmerkeikon og teksten "Nynorsk".
+
+```tsx
+<Chip
```

Comment:
Når en toggle chip er klikket så noterte jeg det skjedde 3 ting:

1. Knappen blir fysisk større for å få plass til checkmarken. Er dette en gjennomtenkt approach? Dette vil føre til at når disse er stacket vertikalt ved siden av hverandre så vil man etter hvert pushe de andre chipene lenger og lenger til venstre eller høyre etter hvor mange chipper som er checked.
2. Knappen har ikke midstilt tekst når den er checked og man hover over
3. Focus state på knappen er ikke rund når den er checked

<img width="678" alt="image" src="https://github.com/digdir/designsystem/assets/31471142/a90ce18c-defb-409e-b131-676d7fd3656e">

<img width="117" alt="image" src="https://github.com/digdir/designsystem/assets/31471142/b281363b-ef1e-4264-b372-67c8b7cc061d">